demos/python_demos/handwritten_japanese_recognition_demo/README.md

```diff
@@ -50,13 +52,27 @@ Options:
                         Optional. Number of inference iterations
   -cl CHARLIST, --charlist CHARLIST
                         Path to the decoding char list file
+  -dc DESIGNATED_CHARACTERS, --designated_characters DESIGNATED_CHARACTERS
+                        Optional. Path to the designated character file
+  -tk TOP_K, --top_k TOP_K
+                        Optional. Top k steps in looking up the decoded
+                        character, until a designated one is found
 ```
 
 
 For example:
 ```
 python handwritten_japanese_recognition_demo.py -i data/test.png -m path/to/ir_xml/model.xml
 
+```
+
+When the `designated_characters` argument is provided, the script will check Top k steps in looking up the decoded character, until a designated one is found. By doing so, the output character will be restricted to a designated region. K is set to 20 by default.
+
+For example, if we want to restrict the output character to a digit and hyphen circumstance, we need to provide the path to the designated character file, then the command line will be like:
+
+```
+python handwritten_japanese_recognition_demo.py -i data/test.png -m path/to/ir_xml/model.xml -dc data/digit_hyphen.txt
```

Comment:
```suggestion
python handwritten_japanese_recognition_demo.py -i data/test.png -m path/to/ir_xml/model.xml -dc data/digit_hyphen.txt -tk 4442
```

Reply:
Such an large `-tk 4442` does not make sense here, as it is a parameter for post-processing.